src/main/java/com/minecraftabnormals/abnormals_core/core/AbnormalsCore.java

```diff
@@ -72,6 +77,7 @@
 	public static final EndimationDataManager ENDIMATION_DATA_MANAGER = new EndimationDataManager();
 	public static final RegistryHelper REGISTRY_HELPER = new RegistryHelper(MODID);
 	public static final TrackedData<Byte> SLABFISH_SETTINGS = TrackedData.Builder.create(DataProcessors.BYTE, () -> (byte) 8).enablePersistence().build();
+	public static final Set<WoodType> WOOD_TYPES = new HashSet<>();
```

Comment:
Not a fan of this field being in `AbnormalsCore`.
A `SignManager` class would be better.